extras/ams_wrapper/src/api/models/model.py

```diff
@@ -14,24 +14,35 @@
 # limitations under the License.
 #
 
+from abc import ABC, abstractmethod
 import datetime
 import json
+import sys
+from typing import Dict
+
 import falcon
 import tensorflow as tf
 import numpy as np
-from abc import ABC, abstractmethod
-from logger import get_logger
+
+from src.logger import get_logger
+from src.api.models.input_config import ModelInputConfiguration, \
+    ModelInputConfigurationSchema, ValidationError
 
 logger = get_logger(__name__)
 
+
 class Model(ABC):
 
-    def __init__(self, ovms_connector):
+    def __init__(self, ovms_connector, config_file_path: str = None):
```

Comment:
Maybe we could already pass inputs configuration here?
Right now, we have model config defined in config.py and it's imported here:
https://github.com/openvinotoolkit/model_server/blob/ams/extras/ams_wrapper/src/api/dispatcher.py#L20
in the process of creating model's objects and dispatcher. 
Perhaps we could extend wrapper parameters to include directory with models configs and then load all of them at the AMS startup. Then just pass configs to appropriate models constructors.